docs/gateway.md

```diff
@@ -88,6 +92,26 @@ As described in [Running Fedimint for dev testing](./dev-running.md#using-the-ga
 - Help add support to other node implementations by building [gateway-lnrpc-extensions](#gateway-lnrpc-extension) for them. You can parent your brand-new extension in this directory, or in your own repository and we will link to it in this open documentation
 - Contributions are highly welcome!
 
+### Developing Gateway-UI (aka, Mintgate)
+
+To interact with and develop Mintgate, open a new terminal and change directories to `/gateway/ui/` and start the development shell
+
+```sh
+cd /gateway/ui
+nix develop .#gateway-ui
+```
+
+Install the required dependencies and run the Mintgate client
+
+```sh
+yarn install
+yarn start
+```
+
+You should see a running Mintgate instance in your local browser. Probably at `https://localhost:3000`
```

Comment:
We should change this port to something in the `81xx` range that we normally use. 3000 likely to cause conflicts.